t/Util.pm

```diff
@@ -525,6 +556,75 @@ sub spawn_h2_server {
     return $server;
 }
 
+package H2OLog {
```

Comment:
How about calling this `H2olog_tracer` or something, and changing `spawn_h2olog` to `new`?

The rationale is that this package is used for running and obtaining traces using h2olog, and that the method is used for instantiating an object that does that.